### PR TITLE
Feat : Integration with OpenMetadata Airflow APIs to make ingestion jobs scheduled through UI.

### LINKED ISSUE #1025: UI: Integration with OpenMetadata Airflow APIs to make ingestion jobs scheduled through UI

### PR DESCRIPTION
**Pull request by Sachin-chaurasiya**
Closes #1025 
### Describe your changes :

I worked on the Ingestion page because of the need to Integrate with OpenMetadata Airflow APIs to make ingestion jobs scheduled through UI.

Features:
- [x] Delete ingestion
- [x] Trigger Ingetion
- [x] Edit Ingestion
- [x] Add Ingestion 
- [x] Ingestion listing

#
### Type of change :

- [x] Improvement
- [x] New feature

#
### Frontend Preview (Screenshots) :
Ingestion Listing Page
image

Add/Edit ingestion Modal

1. Ingestion details
image

2. Connector config
image

3. Scheduling
image

4. Preview and Deploy
image

#
### Checklist:

- [x] I have read the [**CONTRIBUTING**](https://docs.open-metadata.org/open-source-community/developer) document.
- [x] I have performed a self-review of my own. 
- [x] I have tagged my reviewers below.
- [ ] I have commented on my code, particularly in hard-to-understand areas.
- [x] My changes generate no new warnings.
- [ ] I have added tests that prove my fix is effective or that my feature works.
- [x] All new and existing tests passed.

#
### Reviewers
@shahsank3t, @darth-coder00

### CODE REVIEW

File: catalog-rest-service/src/main/resources/ui/src/components/Ingestion/Ingestion.component.tsx [hunk 1]
```diff
@@ -0,0 +1,160 @@
...
+    name: 'BigQuery Ingest',
+    type: 'Usage',
+    service: 'BigQuery',
+    schedule: '1 day, 00:00:00',
+    reecentRuns: ['error', 'success', 'error', 'success', 'error'],
+    nextRun: '2021-11-19 14:00:00',
+  },
+  {
+    name: 'Snowflake Ingest',
+    type: 'Profiler',
```
Comment (harshach) [inline catalog-rest-service/src/main/resources/ui/src/components/Ingestion/Ingestion.component.tsx:23]: there is no type "profiler" and use lower case for type

Comment (Sachin-chaurasiya) [reply catalog-rest-service/src/main/resources/ui/src/components/Ingestion/Ingestion.component.tsx:24]: @harshach sure.

File: catalog-rest-service/src/main/resources/ui/src/components/IngestionModal/IngestionModal.css [hunk 2]
```diff
@@ -0,0 +1,7 @@
+.preview-header {
```
Comment (darth-coder00) [inline catalog-rest-service/src/main/resources/ui/src/components/IngestionModal/IngestionModal.css:1]: Do we need this CSS now?

**Review by darth-coder00 — CHANGES_REQUESTED** [review]
Few files are missing License.
Also added one comment

**Review by darth-coder00 — APPROVED** [review]
LGTM

**Pull request merged**